src/modules/storage/LedgerStorage.ts

```diff
@@ -4730,7 +4744,7 @@ export class LedgerStorage extends Storages {
      * and if an error occurs the .catch is called with an error.
      */
     public getBOAHolders(limit: number, page: number): Promise<any> {
-        const circulating_sql = `select sum(amount) as circulating_supply from utxos`;
+        const circulating_sql = `SELECT circulating_supply FROM blocks_stats WHERE block_height = (SELECT MAX(block_height) FROM blocks_stats)`;
```

Comment:
Can't you combine it with the query below?

Reply:
I don't think there's any common column, If i combine in the query blow it will be a nested query